### DISCUSSION TITLE
why is it only exported for one difficulty

### DISCUSSION THREAD

**Discussion by lijinming2007** [Damnae/storybrew#62] [Q&A] [answered]
I **right-click** to export storyboard, but why is it only exported for **one difficulty**
Like this, these are my code [(link)](https://i.ibb.co/JpL5CgW/1.png)
I know this program changed those osu file, but i don't see any code about this in osu file [(link)](https://i.ibb.co/LQZDKVy/2.png)

**Comment by Damnae** [comment, marked as answer]
In the Layers tab, you need to make that layer diff-specific: 

![](https://puu.sh/IEqiy/cececfedf4.png)
